Sources/Jobs/MemoryJobQueue.swift

```diff
@@ -104,6 +109,13 @@ public final class MemoryQueue: JobQueueDriver {
                     return nil
                 }
                 if let request = queue.popFirst() {
+                    if let delayedJob = self.delayedJobs.removeValue(forKey: request.id) {
+                        if delayedJob > Date() {
```

Comment:
Bit of a nit, but I think it should be `>=`

```suggestion
                        if delayedJob >= Date() {
```